### ISSUE TITLE
Using SMB + authentication on Linux

### ISSUE THREAD

**Issue by Giwayume** [EugeneDae/VLC-Scheduler#33]
I added a SMB folder, vlcscheduler.yaml:

```
sources:
        - path: smb://192.168.1.200/some/folder/
```

This SMB share requires username/password authentication. I don't see where to configure that.

VLC Scheduler just says:
```
Error: The source path is not a directory: smb://192.168.1.200/some/folder/
```

Can you document how this is intended to work with SMB?

**Comment by EugeneDae** [comment]
Can you mount the SMB share as a local folder and use the path to the folder as a `source`?

**Comment by Giwayume** [comment]
Certainly, just wondering if there's a way to do it without mounting. Python has libraries to deal with smb.
